libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/PartitionStat.java

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.openhouse.datalayout.datasource;
+
+import java.io.Serializable;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/** Represents table partition. */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PartitionStat implements Serializable {
```

Comment:
```suggestion
public class TablePartitionStat implements Serializable {
```

Reply:
This class is used in table context always